packages/analytics-js-common/__tests__/utilities/detect.test.ts

```diff
@@ -0,0 +1,18 @@
+import { isSDKRunningInChromeExtension } from '../../src/utilities/detect';
+
+describe('Detect env/feature/capabilities', () => {
+  describe('isSDKRunningInChromeExtension', () => {
+    it('should return true if SDK is running inside chrome extension', () => {
+      (window as any).chrome = {
```

Comment:
Let's undo this change after the test is done, please.
Each test case should have a clean and known start.

Reply:
Done